geomstats/information_geometry/gamma.py

```diff
@@ -0,0 +1,1198 @@
+        point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in natural coordinates.
+
+        Returns
+        -------
+        point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in standard coordinates.
+        """
+        point = gs.to_ndarray(point, to_ndim=2)
+
+        point[:, 1] = point[:, 0] / point[:, 1]
+
+        return gs.squeeze(point)
+
+    def standard_to_natural_point(self, point):
+        """Convert point from standard coordinates to natural coordinates.
+
+        The change of variable is symmetric.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in standard coordinates.
+
+        Returns
+        -------
+        point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in natural coordinates.
+        """
+        return self.natural_to_standard_point(point)
+
+    def natural_to_standard_vec(self, vec, base_point):
+        """Convert tangent vector from natural coordinates to standard coordinates.
+
+        The change of variable is symmetric.
+
+        Parameters
+        ----------
+        base_point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in natural coordinates.
+        vec : array-like, shape=[..., 2]
+            Tangent vector at base_point, given in natural coordinates.
+
+        Returns
+        -------
+        vec : array-like, shape=[..., 2]
+            Tangent vector at base_point, given in standard coordinates.
+        """
+        base_point = gs.to_ndarray(base_point, to_ndim=2)
+
+        n_points = base_point.shape[0]
+
+        kappa, scale = (
+            base_point[:, 0],
+            base_point[:, 1],
+        )
+
+        jac = gs.array(
+            [[gs.ones(n_points), gs.zeros(n_points)], [1 / scale, -kappa / scale**2]]
+        )
+        jac = gs.transpose(jac, [2, 0, 1])
+
+        vec = vec.reshape((n_points, 2, 1))
+        vec = gs.matmul(jac, vec)
+
+        return gs.squeeze(vec)
+
+    def standard_to_natural_vec(self, point):
+        """Convert tangent vector from standard coordinates to natural coordinates.
+
+        The change of variable is symmetric.
+
+        Parameters
+        ----------
+        base_point : array-like, shape=[..., 2]
+            Point of the Gamma manifold, given in standard coordinates.
+        vec : array-like, shape=[..., 2]
+            Tangent vector at base_point, given in standard coordinates.
+
+        Returns
+        -------
+        vec : array-like, shape=[..., 2]
+            Tangent vector at base_point, given in natural coordinates.
+        """
+        return self.natural_to_standard_vec(point)
+
+
+class GammaMetric(RiemannianMetric):
+    """Class for the Fisher information metric on Gamma distributions.
+
+    References
+    ----------
+    Arwini, K. A., & Dodson, C. T. (2008). Information geometry (pp. 31-54).
```

Comment:
This is still not in the same format as e.g. [here](https://github.com/geomstats/geomstats/blob/9bad7b8cc618fc0b9cc3dee13d01698a8f86bb11/geomstats/geometry/spd_matrices.py#L483). It must start with `.. [AD2008]` etc.